search-app/resources/schema/subscription_resource_schema.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
The subscription search result JSON schema should be defined in: [EOSDIS Metadata Format Depot](https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/subscription), e.g. v1.1 is [here](https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/subscription/v1.1/umm-sub-search-results-json-schema.json). Remove this file.

Reply:
done, those changes have been merged there.